test/provisioner/provision_azure_initializer.go

```diff
@@ -0,0 +1,165 @@
+// (C) Copyright Confidential Containers Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package provisioner
+
+import (
+	"errors"
+	"os"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/containerservice/mgmt/containerservice"
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/network/mgmt/network"
+	"github.com/Azure/azure-sdk-for-go/profiles/latest/resources/mgmt/resources"
+	"github.com/Azure/go-autorest/autorest/azure/auth"
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+	initAzureLogger()
+}
+
+func initAzureLogger() {
+	level := os.Getenv("LOG_LEVEL")
```

Comment:
I wonder if the library automatically picks up the value and we don't have to do this switch case?